src/Wizard.js

```diff
@@ -34,6 +37,23 @@ const Wizard = ({ children, onComplete, debug }: Props) => {
     ]);
   }
 
+  function updateHistory(currStepName, nextStepName) {
+    if (typeof window === 'undefined') {
+      return;
+    }
+    const currUrl = window.location.href;
+    const basePath = currUrl.includes('?') ? currUrl.split('?')[0] : currUrl;
+    const searchParams = new URLSearchParams(new URL(currUrl).searchParams);
+    searchParams.set('step', nextStepName);
+
+    window.history.pushState({ activeStep: currStepName }, '', currUrl);
+    window.history.replaceState(
```

Comment:
What is happening here? Why do we need both `pushState` and `replaceState`?

Reply:
.pushState() adds the current url to the browser's history stack.
.replaceState() updates the url to the new url, without reloading the page.

Once these methods are run, the user will be able to use the browser backwards and forwards functions.